docs/Tutorials/Python.md

```diff
@@ -26,7 +26,12 @@ supports `pip`, for instance in a
 [Anaconda](https://www.anaconda.com/distribution/) environment. You can also get
 access to the SpECTRE Python modules by adding  `BUILD_DIR/bin/python` to your
 `PYTHONPATH`. This is done automatically by sourcing the `LoadPython.sh` file
-with the command `. BUILD_DIR/bin/LoadPython.sh`.
+with the command `. BUILD_DIR/bin/LoadPython.sh`. By default, SpECTRE uses
+`jemalloc` which needs to be pre-loaded for the python bindings to work.
+Therefore, you need to run `LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so
```

Comment:
This path can be different on each system, could you make that clear?

Reply:
ah right, I just saw this was an environment variable for the cmake output. I suppose there is no universal way to find the path. how about `LD_PRELOAD=/path/to/libjemalloc.so python` ?

Reply:
Works for me. They can always fall back to the system allocator.